src/core_plugins/kibana/public/dashboard/panel/panel_header.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { KuiKeyboardAccessible } from 'ui_framework/components';
+import { PanelOptionsMenu } from './panel_options_menu';
+
+export class PanelHeader extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getOptionsDropDown() {
+    return (
+      <PanelOptionsMenu
+        onEditPanel={this.props.onEditPanel}
+        onDeletePanel={this.props.onDeletePanel}
+        onToggleExpandPanel={this.props.onToggleExpand}
+        isExpanded={this.props.isExpanded}
+      />
+    );
+  }
+
+  getExpandToggle() {
+    const { isExpanded } = this.props;
+    const classes = classNames('kuiIcon', null, { 'fa-expand': !isExpanded, 'fa-compress': isExpanded });
+    return (
+      <KuiKeyboardAccessible>
+        <a
```

Comment:
Could you make this a `button` instead and just style it whatever way you like? This is an action and not a navigation to another page, so it should rather be a `button` not an `a` (also you could get rid of the `KuiKeyboardAccessible` element around it in this case.